plugins/chat/assets/javascripts/discourse/services/chat-channels-manager.js

```diff
@@ -111,12 +112,17 @@ export default class ChatChannelsManager extends Service {
 
   @cached
   get publicMessageChannels() {
-    return this.channels
-      .filter(
-        (channel) =>
-          channel.isCategoryChannel && channel.currentUserMembership.following
-      )
-      .sort((a, b) => a?.slug?.localeCompare?.(b?.slug));
+    const myChannels = this.channels.filter((channel) => {
+      return (
+        channel.isCategoryChannel && channel.currentUserMembership.following
+      );
+    });
```

Comment:
```suggestion
    const channels = this.channels.filter((channel) => channel.isCategoryChannel && channel.currentUserMembership.following);
```

This is not going to be correctly formatted, but you get the idea